src/app-layer-htp.h

```diff
@@ -257,10 +257,14 @@ typedef struct HtpState_ {
     uint16_t events;
     uint16_t htp_messages_offset; /**< offset into conn->messages list */
     uint32_t file_track_id;             /**< used to assign file track ids to files */
+    uint32_t file_range_ids;             /**< used to assign track ids to range file */
     uint64_t last_request_data_stamp;
     uint64_t last_response_data_stamp;
 } HtpState;
 
+/** Flag to track reassembled file from multiple ranges */
+#define FLAG_FILE_RANGES 0x80000000
```

Comment:
We trust there will be no overflow...

Reply:
I don't see how this is used. I mean I see it is being set, but thats all.

Reply:
The point is to have an id different than the other files.
But maybe we can have this different id without using this flag, just incrementing the id for the next file...
What do you prefer ?

Reply:
If we can do without 'tricks', I prefer that.